Strategy11/business-directory-plugin
# Introduce Floating Links in the Business Directory Panel

This PR introduces the Floating Links feature to the Business Directory Panel, which provides users with a set of links for quick navigation to the **Upgrade**, **Support**, and **Documentation** pages.

#### NOTE: To test the `PRO version links`, activate the BD PRO plugin.

## Testing Instructions:
1. Navigate to `WP Admin > Directory`.
2. Check the perfect pixel implementation.
  - **Figma File:** https://www.figma.com/file/YcSWkrjVGroC1zyxhuHH6W/Floating-Links?node-id=1-117&t=Xf5EKO8Mvs1JwuFj-4
3. Verify that all functionality works well.

## Demo - BD Free Version:
![2023-07-03 16 16 19](https://github.com/Strategy11/business-directory-plugin/assets/69119241/43d6f610-dd14-4d0a-b662-1361b77add3a)

## Demo - BD PRO Version:
![2023-07-03 16 29 20](https://github.com/Strategy11/business-directory-plugin/assets/69119241/df836213-4759-4638-9244-e651a02daa79)
